README.md

```diff
@@ -73,6 +76,25 @@ This command is deprecated in favor of `ember try:one`. There are several bugs w
 
 ### Config
 
+##### versionCompatibility
+If you're using `ember-try` with an Ember addon, there is a short cut to test many Ember versions. In your `package.json` under the `ember-addon` key, add the following: 
+
+```json
+  "ember-addon": {
+    "versionCompatibility": {
+       "ember": ">1.11.0 <=2.0.0"
```

Comment:
Last comment: can you extend that to support "default" "beta" and "canary"? This is close to the situation where we don't need to include an ember-try config by default.